### ISSUE TITLE
[Feature Request]: Make adding Startup Apps easier

### ISSUE THREAD

**Issue by youdontneedtoknow22** [pop-os/gnome-shell-extension-pop-cosmic#246]
There's a way around this, but I believe this should be done easier.
I'm using Mega for syncing and I noticed that it's not starting up automatically, so I wanted to add it. In windows it's easy, you click add and choose the path (C:/program files/mega/..), in Linux it's a lot more complicated with Flatpaks and apt (especially when you still don't have an idea where are the programs installed).
I couldn't find the path nor the command for Mega (without installing a 3rd party software)

Can you integrate "choose from installed apps" to the Startup Applications utility? Right there:
![image](https://user-images.githubusercontent.com/85135068/142930005-68703cf6-538c-419b-8985-30cab7a54ed8.png)

**Comment by JonathanSundberg** [comment]
Bump +1
Linux mint also has a much easier way to add startup progams, not sure why pop os requires a command and everything

**Comment by Meek97** [comment]
I agree, this process should be a little more streamlined, especially for users who aren't as comfortable with terminal commands. If someone has no idea about flatpak / debian packages, and is just using the pop-shop to install apps this interface would be totally alien to them. The inclusion of some sort of app-picker would be a boon to that type of user, and save more savvy users from tracking down app ID's and doing a few google searches.

For anyone else who ends up here looking for a solution, here's the TL;DR for flatpak apps at least:
get the flatpak app id, and use the following command format `flatpak run <flatpak app ID>` in Startup Applications

[How to start a flatpak at boot?](https://www.reddit.com/r/pop_os/comments/mslxjx/how_to_start_a_flatpak_at_boot/)
> [marvelggg](https://www.reddit.com/user/marvelggg/)  [3y ago](https://www.reddit.com/r/pop_os/comments/mslxjx/comment/guvw5ch/) Edited 3y ago
>
>Search for the Application ID:
`flatpak search deluge`
then use this program to run it:
`flatpak run org.deluge_torrent.deluge`
...
